### PR TITLE
Akupara ship and guide

### PR DESCRIPTION
**Pull request by tonotom1**
## About the PR
I made a cool botany ship with wacky atmos and a small chem lab with a biodome / park theme and added a guide entry for it

The turtle shape was an accident but I roll with it because it is cool

## Why / Balance
I wanted a cool botany ship that could do atmos and mutations out the gate! I tried balancing the fact that it is more kitted than the Garden by using mostly dirt instead of hydroponics trays

## How to test
1: Use mapping console command and then loadgrid akupara.yml, or buy the ship at the shipyard console
2: look at it how cool it is!!!!

## Media
image

TORTL!!
image

## Breaking changes
I messed with the guidebook files too which looks less flexible to change than the addition of the ship itself. Exclude guidebook changes if they may cause interference.

**Changelog**
-add: Akupara, a medium full-feature botany ship to the Frontier shipyard console intended to provide a platform to experience all aspects of botany possible out the gate, but not necessarily be the best compared to a custom solution.

### CODE REVIEW

**Comment by dvir001** [comment]
Before I even talk about issues to fix.

So this ship is just build to farm as much money from gas mutation?

That's it?

How is this MRP if you just going to hermit in this ship just farming money with gas mutations?

**Comment by tonotom1** [comment]
> Before I even talk about issues to fix.
> 
> So this ship is just build to farm as much money from gas mutation?

The gas harvesting aspect is present specifically because I thought of everything that botany could do and tried to design something where it could interact with every situation that could arise. The atmos system was also because I was also inspired to build an atmos system that involved more than just air in waste out.

> That's it?

No, there's the all-encompassing aspect of botany itself. I love botany. I love hearing the plants scream and seeing them change sprites when I put mutagen in them. I love seeing dying plants brought back from the brink of my abuse and triumphing over the Kudzu trying to kill my cool mutant plants. I love handing out all manner of liquids, helpful and novelty fresh fresh from the grinder to people that have never even heard of their names. I love seeing the screen sparkle and fog with messed up gases! Speaking of gases, I also want people tinkering with the atmos system because I know for a fact what I designed was not a perfect system. I love seeing people walk into my garden and then slip because the entire ground is covered in space lube and slippery mutated fruit! I love hearing my visitors' opinions and observations and being astounded at the plants sticking out of my tray!!

> How is this MRP if you just going to hermit in this ship just farming money with gas mutations?

Because I'm not. I got sick of repeatedly tiding the Liquidator for its chem master and chem supplies to be allowed to have fun with plants and sick of being only holed up in the 1-tile wide tray bent tube that is the Garden. If literally any other full-feature enough botany vessel was added, I could buy it once and not have to scum a second ship. If cash brainrot was my goal, a similar level of performance can theoretically be achieved with the Garden by just popping the passive waste vent off the Garden and replacing it with a few connector ports: something I casually tested, but have not fallen into the pitfall of narrowly chasing with tunnel vision. My botany projects have always been at a loss, or purely subsidized by the kindness of other players. This is genuinely something I love performing for other players to see and benefit from.

File: Resources/Maps/_NF/Shuttles/akupara.yml [hunk 1]
```diff
@@ -0,0 +1,4264 @@
...
+    - type: Physics
+      bodyType: Static
+- proto: PestSpray
+  entities:
+  - uid: 362
+    components:
+    - type: Transform
+      pos: -0.5,2.5
+      parent: 1
+- proto: PortableGeneratorSuperPacmanShuttle
```
Comment (dvir001) [inline Resources/Maps/_NF/Shuttles/akupara.yml:2921]: This generator getting rads added at some point
Either make it fully closed in a rads protection chamber with full locker kit for rads, or replace with plasma generators

Also use shuttle generators.

File: Resources/Maps/_NF/Shuttles/akupara.yml [hunk 2]
```diff
@@ -0,0 +1,4264 @@
...
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-12.5
+      parent: 1
+  - uid: 376
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-8.5
+      parent: 1
+- proto: PuddleVomit
```
Comment (dvir001) [inline Resources/Maps/_NF/Shuttles/akupara.yml:3021]: Dont add any mapped trash or puddles, it gets added with automation.

File: Resources/Maps/_NF/Shuttles/akupara.yml [hunk 3]
```diff
@@ -0,0 +1,4264 @@
...
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -3.5,5.5
+      parent: 1
+  - uid: 396
+    components:
+    - type: Transform
+      pos: 4.5,0.5
+      parent: 1
+- proto: RandomVendingDrinks
```
Comment (dvir001) [inline Resources/Maps/_NF/Shuttles/akupara.yml:3147]: Remove

File: Resources/Maps/_NF/Shuttles/akupara.yml [hunk 4]
```diff
@@ -0,0 +1,4264 @@
...
+      pos: 2.5,-0.5
+      parent: 1
+    - type: DeviceNetwork
+      configurators:
+      - invalid
+      deviceLists:
+      - 2
+    - type: AtmosPipeColor
+      color: '#FFAA00FF'
+- proto: GasVolumePump
```
Comment (dvir001) [inline Resources/Maps/_NF/Shuttles/akupara.yml:2387]: Main air setup best to be on on startup with the mixer if possible.

**Review by dvir001 — CHANGES_REQUESTED** [review]

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 2 above)
Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:3021]: Are you referring to the trash spawners too?

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 4 above)
Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:2387]: I smuggled that change in already after your feedback in the shipbuilding channel hehe

Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:2387]: Addendum, thank you for the timely feedback. I am leaving my country imminently so I won't be able to make changes until I am cozy across the globe.

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 2 above)
Comment (dvir001) [reply Resources/Maps/_NF/Shuttles/akupara.yml:3021]: Yes, they are automated now so they are not required or needed anymore.

**Comment by dvir001** [comment]
Keep your changelog format as it need to be
One line, explain the name of the ship, short note on it and what shipyard its on.

File: Resources/ServerInfo/_NF/Guidebook/Shipyard/Akupara.xml [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  - First, disconnect any airlocks that are connected to the dock.
+  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with [color=#028ed9]"W"[/color] and [color=#028ed9]"S"[/color] being forward and backward, [color=#028ed9]"A"[/color] and [color=#028ed9]"D"[/color] being left and right, and [color=#028ed9]"Q"[/color] and [color=#028ed9]"E"[/color] being rotating left and right; and [color=#028ed9]"spacebar"[/color] being the brake and moving precicely by holding the [color=#028ed9]"spacebar"[/color] while doing other inputs.
+  
+  # Hiring crew
+  <Box>
+    <GuideEntityEmbed Entity="ComputerStationRecords"/>
+    <GuideEntityEmbed Entity="ClothingHeadsetGrey"/>
+    <GuideEntityEmbed Entity="IntercomCommon"/>
+  </Box>
+  As a Captain of a Nanotrasen vessel, you have the authority to hire, fire, demote, or promote crew members at will. There are two ways you as a Captain can go about hiring crew: you can either use the [color=#a4885c]station records computer[/color] on your ship to open crew slots (jobs available vary from ship to ship) or invite through in-game communications other players to your crew.
```
Comment (ErhardSteinhauer) [inline Resources/ServerInfo/_NF/Guidebook/Shipyard/Akupara.xml:43]: Sections for `Piloting` and `Hiring crew` no longer needed - those were moved to a separate guidebook entries

File: SpaceStation14.sln
Comment (ErhardSteinhauer) [inline SpaceStation14.sln:1]: This file shouldn't be included in your PR

File: Resources/Prototypes/_NF/Guidebook/shipyard.yml [hunk 6]
```diff
@@ -5,6 +5,7 @@
   priority: 4
   children:
   - ShipyardAmbition
+  - ShipyardAkupara
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Guidebook/shipyard.yml:8]: Not a strict requirement, but try to keep it in alphabetical order

**Review by ErhardSteinhauer — CHANGES_REQUESTED** [review]

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 3 above)
Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:3147]: Would a non-random drink vendor (IE Starkist vendor) be acceptable? It's fine if not, it's mainly fluff just to tie the park bench theme.

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 2 above)
Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:3021]: All clutter removed.

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 1 above)
Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml:2921]: Double checked generator type used, confirmed to be the shuttle variant. Generator enclosure adjusted to add radiation shutter on access side.

File: Resources/Prototypes/_NF/Guidebook/shipyard.yml (hunk 6 above)
Comment (tonotom1) [reply Resources/Prototypes/_NF/Guidebook/shipyard.yml:8]: Oops, off by one spot. Fixed.

File: Resources/ServerInfo/_NF/Guidebook/Shipyard/Akupara.xml (hunk 5 above)
Comment (tonotom1) [reply Resources/ServerInfo/_NF/Guidebook/Shipyard/Akupara.xml:43]: Guide redundancy removed.

File: SpaceStation14.sln
Comment (tonotom1) [reply SpaceStation14.sln:1]: I tried to figure out how to exclude this file in my PR and have attempted to delete the file. Notify if this has not resolved the issue.

File: Resources/Maps/_NF/Shuttles/akupara.yml (hunk 3 above)
Comment (dvir001) [reply Resources/Maps/_NF/Shuttles/akupara.yml]: No food / drink vends

Comment (tonotom1) [reply Resources/Maps/_NF/Shuttles/akupara.yml]: Adjustment made according to guideline clarifications

**Review by MagnusCrowe — CHANGES_REQUESTED** [review]
image
Your disposals kills people.
Don't use custom filled crates/lockers. (no free borg parts)
Your engineering locker and captain's locker seem to be open by default.
You do not need a super pac man.

image
There's no drain in the chem lab.
Having a plant right infront of the chem dispenser makes no sense.

image
I don't understand why there would be hydroponic trays and soil plots...probably bet to just go with soil.

image
I don't like that you're mixing two different wall/window styles.  Pick one.

image
Use red airlock doors instead of these normal airlock doors.

image
Avoid putting pipes in walls at as much as possible.

image
This half of the ship could benefit from being bumped out one more tile, wider.  It would make the docking port normal sized, give more room for piping things so they aren't in walls, and give you room for more chemistry machines.  I feel like the ship is focused on producing gases through botany so having a condenser and electrolyzer in your chem lab to extract chems from gases seems like a good idea.

image
Seems weird to me to need to walk through atmos to access most of the ship...IDK what you can do about it, just very weird.  There's not even any catwalks to walk on.


What would you do with this ship if we took away botany gases?  Would it still be relevant?

I like that it is asymmetric. I like the garden area, it's very fancy but it's strange that it's so nice on one area and so industrial in the rest of the ship.  There's a weird dissonance, especially because your entrance to the posh, nicely decorated area, is an airlock in a atmos room.

**Comment by tonotom1** [comment]
Cool feedback, let's see what I can do with it!

I played into the asymmetric nature by theming the halves as nice on one side and backrooms the other. I imagined it being a dome cradled by its system-critical and supporting equipment all on the other end. I think I can toy with the airlock position to make the dome the "central walkway" rather than the atmos room easily though!

Excellent thought on the addition of that chem equipment, by the way. I originally thought of adding those too but thought it was too "loaded and overkitted". However with your feedback considered I feel emboldened that they can be included and I reckon I can justify the usage of the SUPERPAC with the additional power draw.

Also regarding the ship's relevancy without botany gases: The ship's design is a reflection of every interaction currently possible with botany and if that is altered then it be what it be. An "everything botany" ship would be entirely different and will have to perform different.

**Comment by tonotom1** [comment]
Also I would like to reiterate that the main focus is not atmos farming, but a ship capable with interacting with the plants in every way possible. The atmos isn't even optimized for it (I made sure to design it to do a bare minimum job) and is primarily designed to be configurable to control plant growing conditions in the incident oxygen or nitrogen ruin things via overpressure. It just has the feature added in as a side-thought. You'd really have to go out of your way to rip out the plumbing to actually be able to compress the gases viably for any meaningful financial gain and it'd still likely come out to a petty sum.

If I'm not mistaken, botany atmos farming hasn't been a problem in ages. People seem to just capture gas on expeds nowadays.

**Comment by tonotom1** [comment]
Here's what it looks like after feedback!

**Comment by tonotom1** [comment]
Closing due to conflicts that must be resolved, and due to repo switching struggles in visual studio

**Pull request closed without merging**